sound/soc/soc-topology.c

```diff
@@ -603,10 +603,11 @@ static int soc_tplg_kcontrol_bind_io(struct snd_soc_tplg_ctl_hdr *hdr,
 				sbe->get = ext_ops[i].get;
 		}
 
-		if (sbe->put && sbe->get)
-			return 0;
-		else
+		if ((k->access & SNDRV_CTL_ELEM_ACCESS_TLV_READ) && !sbe->get)
```

Comment:
@RDharageswari the patch looks good now but can you please simplify the commit message to say something along these lines. There's no need to mention anything about volatile here as you're not really checking that here.

"Add support for RO/WO TLV byte controls

Add support for read-only and write-only byte controls by checking for the appropriate get/put IO handlers"

Reply:
Done